### PR TITLE
Adding MKLDNN benchmark ai scripts

### PR DESCRIPTION
**Pull request by mseth10**
Moving benchmark ai scripts from personal repo to aws repo.

### CODE REVIEW

**Comment by mseth10** [comment]
Please review!
@apeforest @anirudh2290 @piyushghai @lanking520 @frankfliu @leleamol

File: image_classification/infer_imagenet.py [hunk 1]
```diff
@@ -0,0 +1,198 @@
+iimport argparse, time, logging, os
```
Comment (stu1130) [inline image_classification/infer_imagenet.py]: nit:
```suggestion
import argparse, time, logging, os
```

File: word_language_model/lstm_bucketing_infer.py [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+        idx2word[val] = key
+
+    # Load pre-trained lstm bucketing module
+    num_layers = 2
+    num_hidden = 200
+    num_embed = 200
+
+    stack = mx.rnn.FusedRNNCell(num_hidden, num_layers=num_layers, mode="lstm").unfuse()
+
+    mxnet_ctx = mx.cpu()
```
Comment (piyushghai) [inline word_language_model/lstm_bucketing_infer.py]: I assume it is intentional to run it on CPU always ?

File: image_classification/image_classification.py [hunk 3]
```diff
@@ -181,13 +183,20 @@ def train(epochs, ctx, dtype):
...
         softmax = mx.sym.SoftmaxOutput(out, name='softmax')
         mod = mx.mod.Module(softmax, context=[mx.gpu(i) for i in range(gpus)] if gpus > 0 else [mx.cpu()])
-        mod.fit(
-            train_data,
-            val_data,
-            num_epoch=opt.epochs,
-            kvstore=opt.kvstore,
-            batch_end_callback=mx.callback.Speedometer(batch_size, opt.log_interval)
-        )
+        if opt.infer:
```
Comment (apeforest) [inline image_classification/image_classification.py]: Why do you need an option here if you already created a infer_imagenet.py script in the same folder?

Comment (apeforest) [inline image_classification/image_classification.py]: Why do you need this option here if you already added an infer_imagenet.py script

File: image_classification/infer_imagenet.py [hunk 4]
```diff
@@ -0,0 +1,198 @@
+import argparse, time, logging, os
```
Comment (apeforest) [inline image_classification/infer_imagenet.py]: What's this file for?

File: image_classification/image_classification.py (hunk 3 above)
Comment (mseth10) [reply image_classification/image_classification.py]: Right, I don't need infer in this file. Removing it. Thanks for pointing it out.

File: image_classification/infer_imagenet.py (hunk 4 above)
Comment (mseth10) [reply image_classification/infer_imagenet.py]: This file is for inference with resnet50 on imagenet dataset. I have removed the infer code from the other file.

Comment (apeforest) [reply image_classification/infer_imagenet.py]: After offline discussion, this file is more update than predict_image.py in the same folder. We should keep this one and replace predict_image.py (after making sure no other script uses it) in the future to keep the repo clean.

File: word_language_model/lstm_bucketing_infer.py (hunk 2 above)
Comment (mseth10) [reply word_language_model/lstm_bucketing_infer.py]: Yes, it is meant to be run on CPU only. The script runs inference with MKLDNN.

**Pull request merged**